Searching for the taxonomic classification of the protein JIP3: finding its subclades.

All information and work done with this assignment will be applied to classifying the scaffolding protein JIP3 and its close relatives and familial history. Information will be taken from already available sources like BLAST. The reference figure is taken from Dietmar Kültz, Evolution of Osmosensory MAP Kinase Signaling Pathways, American Zoologist. 
Introduction/Background 


JIP3 comes from the JIP family of MAPK scaffolding proteins. MAPK signaling affects axon susceptibility to pathological degeneration, yet the specific ways MAPK controls these outcomes are unknown. The JIP proteins have a scaffolding role in MAPK signaling. Little is known about the JIP scaffolding proteins, but JIP3, when inhibited, is found to slow down the degeneration of axons in response to injury or chemotherapy-induced neuropathy. Due to the need for more information on the function of the JIPs, constructing a taxonomic classification of the proteins would be a useful source of information. Relatives of the proteins or proteins with similar sequences could provide an example of a system/process similar to axonal degeneration, which could be used as a comparison to the proteins to model and explore their contribution to the degeneration process.

To document and visualize the subclades and relatives of JIP3, I want to create a phylogenic tree that will include my protein of interest, JIP3, and a further curated list of MAPK kinases I think are related to JIP3 and could help to develop a better understanding of its degenerative function. The remaining proteins of interest include JIP1, JIP2, JIP4, stathman2, and SARM1. The paper Evolution of Osmosensory MAP Kinase Signaling Pathways shows a study involving insight into MAP kinase networks and how they function within cells. Kültz aligned the MAP kinase sequences and constructed a tree based on their analysis to understand various motifs across eukaryotes and prokaryotes.

**Insert Image here**
Fig 2. Radial phylogenic tree of MAP kinases from C. Elegans and H. Sapiens. Side B shows paralogous MAP kinases in Homo Sapiens, and side A shows the same in Caenorhabditis elegans. The human tree shows examples of subfamilies within JNK whose activation is facilitated by the JIP proteins. The alignment and analysis process to create this comparison was meant to provide input sa to MAP kinase’s previous signal transduction activity. 
Reference 
Dietmar Kültz, Evolution of Osmosensory MAP Kinase Signaling Pathways, American Zoologist, Volume 41, Issue 4, August 2001, Pages 743–757, https://doi.org/10.1093/icb/41.4.743
Expected Figure 

The expected figure will look similar to fig 2 but in a linear, cladogram form. If my initial run with my selected query sequence is successful, I plan to produce more of a similar figure for other proteins studied in the lab or expand my first figure to include more distant relatives. 
**Insert image here**
Ideal for the final figure template, possibly without the root. 

Materials and Methods

The first step would be to find a reference tree: I will search background literature for a characterization of the JIP family in order to describe the gene’s relations to humans and which classes they belong to. From this information, I will select the best reference(s) to put through BLAST, finding other inputs related to my query. Using either BLAST’s tree visualization option or a method mixed with Linux tools, I will develop an output that shows the subclades of the family to which JIP3 belongs. 
